packages/api-reference/src/components/Content/Introduction/Introduction.vue

```diff
@@ -25,6 +25,10 @@ const props = defineProps<{
 const specVersion = computed(() => {
   return props.parsedSpec.openapi ?? props.parsedSpec.swagger ?? ''
 })
+
+const formattedSpecTitle = computed(() => {
+  return props.info.title?.toLowerCase().replace(/\s+/g, '-')
```

Comment:
We could also use the github-slugger, it’s in the dependencies already: https://github.com/scalar/scalar/blob/7fb8d1285b8b11503effa7d067ceca5fd0e86176/packages/api-reference/src/helpers/getHeadingsFromMarkdown.ts#L3

Wdyt?

Reply:
good call!